Epilepsy/epilepsy_utils.py

```diff
@@ -0,0 +1,119 @@
+from collections import defaultdict
+from itertools import product
+from string import punctuation
+
+from pandas import DataFrame
+
+def offsets_to_token(left, right, offset_array, lemmas, punc=set(punctuation)):
```

Comment:
FYI, `punc=set(punctuation)` can be dangerous. From http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments:

> Python’s default arguments are evaluated once when the function is defined, not each time the function is called (like it is in say, Ruby). This means that if you use a mutable default argument and mutate it, you will and have mutated that object for all future calls to the function as well.

Probably doesn't matter here, unless you ever modify `punc`.